projects/ngx-charts-on-fhir/src/lib/fhir-chart-legend/sort-by.pipe.ts

```diff
@@ -0,0 +1,13 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { Dataset } from '../data-layer/data-layer';
+import { byMostRecentValue } from '../fhir-chart-summary/statistics.service';
+
+@Pipe({ name: 'sortBy' })
+export class SortByPipe implements PipeTransform {
```

Comment:
Let's rename this now that behavior is more specific. How about `sortDatasets`?

Reply:
Okay @semicolin